From 35f4fd54f4d93c9194583ab9678fe16c679b7978 Mon Sep 17 00:00:00 2001
From: tista500 <tista.gma500@gmail.com>
Date: Fri, 6 Oct 2017 22:44:17 +0900
Subject: [PATCH] Gtk-3.22&4.0: [GtkWindow] Stop setting max'd/fullscreen'd
 specific shadows

I'm not sure but this may or may not be related to the issue: #527.
---
 gtk/sass/3.22/_common.scss    | 6 +-----
 gtk/sass/4.0/_common.scss     | 6 +-----
 gtk/sass/common/_drawing.scss | 4 ----
 3 files changed, 2 insertions(+), 14 deletions(-)

diff --git a/gtk/sass/3.22/_common.scss b/gtk/sass/3.22/_common.scss
index 18195f33e..dd70eb02a 100644
--- a/gtk/sass/3.22/_common.scss
+++ b/gtk/sass/3.22/_common.scss
@@ -5906,11 +5906,7 @@ decoration {
 
   .fullscreen &,
   .maximized & {
-    // draw empty shadows when max'd and fullscreen'd
-    &, &:backdrop {
-      border-radius: 0;
-      box-shadow: $z-depth-2-backdrop-maxd;
-    }
+    &, &:backdrop { border-radius: 0; }
   }
 
   @each $_constraint,
diff --git a/gtk/sass/4.0/_common.scss b/gtk/sass/4.0/_common.scss
index 6c026d429..e2afcf0bb 100644
--- a/gtk/sass/4.0/_common.scss
+++ b/gtk/sass/4.0/_common.scss
@@ -5798,11 +5798,7 @@ decoration {
 
   .fullscreen &,
   .maximized & {
-    // draw empty shadows when max'd and fullscreen'd
-    &, &:backdrop {
-      border-radius: 0;
-      box-shadow: $z-depth-2-backdrop-maxd;
-    }
+    &, &:backdrop { border-radius: 0; }
   }
 
   @each $_constraint,
diff --git a/gtk/sass/common/_drawing.scss b/gtk/sass/common/_drawing.scss
index e3796904c..ee6b13ef5 100644
--- a/gtk/sass/common/_drawing.scss
+++ b/gtk/sass/common/_drawing.scss
@@ -42,8 +42,6 @@ $z-depth-5: 0 8px 10px -4px rgba(0, 0, 0, 0.20),
 $z-depth-1-backdrop: 0 2px 2px transparent,
                      0 3px 4px transparent,
                      0 3px 3px -1px rgba(0, 0, 0, 0.08);
-$z-depth-1-backdrop-maxd: 0 2px 2px transparent,
-                          0 3px 4px transparent;
 $z-depth-1-backdrop-tiled: 0 2px 2px transparent,
                            0 3px 4px transparent,
                            1px 0 $panel_bg_color, // side-edges
@@ -65,8 +63,6 @@ $z-depth-2-backdrop: 0 6px 8px -4px transparent,
                      0 10px 6px transparent,
                      0 2px 2px rgba(0, 0, 0, 0.12),
                      0 3px 4px rgba(0, 0, 0, 0.20);
-$z-depth-2-backdrop-maxd: 0 6px 8px -4px transparent,
-                          0 10px 6px transparent;
 $z-depth-2-backdrop-tiled: 0 6px 8px -4px transparent,
                            0 10px 6px transparent,
                            1px 0 $panel_bg_color,
